cvIgnacioCalahorro.html

```diff
@@ -89,30 +83,28 @@ <h2 class='about'>ABOUT ME:</h2>
                 <p class="ses">Studies</p>
                 <div class="texto-justificado">
                     <p class="unis"> 
-                        Msc GIS at Esri Spain 
+                        MSc in GIS at Esri Spain 
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Graduated expected on May 2022
+                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Expected to graduate in May 2022
```

Comment:
- "Graduated expected" is grammatically wrong, either "Graduation expected" or "Expected to graduate". The latter sounds a little bit better in my opinion.
- "on" -> "in". When talking months, it's "in", "on" is used for days (e.g. on Tuesday I got so fucked up I felt like Jorgito on the next day)